src/pat/validation/documentation.md

```diff
@@ -68,6 +68,7 @@ Error messages can also be overridden on a per-field basis, for example:
 | Property          | Description                                                                        | Default | Type |
 |-------------------|------------------------------------------------------------------------------------|---------|------|
 | disable-selector  | A selector for elements that should be disabled when there are errors in the form. |         | CSS Selector |
+| equality          | Field-specific. The name of another input this input should equal to (useful for password confirmation). | | String |
```

Comment:
Please also add an entry for ``message-equality``.

Reply:
Added!